pkg/kv/dist_sender_server_test.go

```diff
@@ -2094,6 +2094,44 @@ func TestTxnCoordSenderRetries(t *testing.T) {
 			},
 			txnCoordRetry: true,
 		},
+		{
+			name: "write too old with initput",
+			afterTxnStart: func(ctx context.Context, db *client.DB) error {
+				return db.Put(ctx, "iput", "put")
+			},
+			retryable: func(ctx context.Context, txn *client.Txn) error {
+				return txn.InitPut(ctx, "iput", "put", false)
```

Comment:
this and all the follow tests should be also tried out with failOnTombstone flag set to true

Reply:
why does this fail, the values are both equal to "put"

Reply:
Good catch. We weren't checking that the failure had to occur in the unittest and it shouldn't have in this case. Fixed now. I added a `failOnTombstones=true` case, but it's not very interesting for these retry tests.